example_configs/prod/mkw_ct_prod_config.json

```diff
@@ -1,51 +1,41 @@
 {
     "token": "",
+    "is_production": true,
     "lounge": "MKW",
     "lounge_description": "Valid options are MKW or MK8DX",
     "guild_id": 387347467332485122,
-    "admin_roles": {
-        "387347467332485122": [
-            "Administrator",
-            "Lounge Staff"
-        ]
-    },
     "queue_join_channel": 1253109700107567155,
     "queue_sub_channel": 1253109752418930779,
     "queue_list_channel": 1253109466027655338,
     "queue_history_channel": 1253109800196243618,
     "queue_general_channel": 1253109398193045534,
-    "members_for_channels": {
-        "387347467332485122": [
-            235148962103951360,
-            155149108183695360,
-            815483289515196416,
-            892420397570592768,
-            775415990238314518,
-            774812906830299136,
-            745443073202323586,
-            774866940950872095,
-            1019051989734273054
-        ]
-    },
-    "roles_for_channels": {
-        "387347467332485122": [
-            520808645252874240,
-            520808674411937792
-        ]
-    },
-    "staff_roles": [
-            387347888935534593,
-            399382503825211393,
-            399384750923579392,
-            521149807994208295,
-            792891432301625364,
-            877629421811540016
+    "admin_roles": [
+        387347888935534593,
+        877629421811540016
```

Comment:
In addition to the staff helper role, i wanted some kind of role that says that someone has authority to use commands, this doesnt come up that often because you would just set integrations within discord itself for what roles can use slash commands, i think the only place its used was some stupid request for some staff to be able to use the sub command without cooldown, and i used this for mk8 for the schedule_sq_times command which isnt a slash command.  

I think those roles are the Boss role and the Developer role for mkwii.